src/main/resources/reference.conf

```diff
@@ -1145,3 +1145,7 @@ distributedLock {
 }
 
 blockedEmailDomains = ["qwiklabs-gsuite.net", "qwiklabs.net"]
+
+termsOfService {
+  enableGracePeriod = false
```

Comment:
to be enabled as part of https://github.com/broadinstitute/firecloud-develop/pull/2753 once the grace period goes live